packages/studio/src/components/Viewport/defaults.tsx

```diff
@@ -0,0 +1,88 @@
+export interface Viewport {
+  name: string;
+  styles: { height: number; width: number } | null;
+  type: "desktop" | "mobile" | "tablet" | "other";
+  css: string;
+}
+
+export interface ViewportStyles {
+  name: string;
+  height: number;
+  width: number;
+}
+
+export interface ViewportMap {
+  [key: string]: Viewport;
+}
+
+export const EXPANDED_VIEWPORTS: ViewportMap = {
```

Comment:
Small nit: Could we rename `EXPANDED_VIEWPORTS`? I think the term "Expanded" doesn't make too much sense now that we've removed "Minimal".